scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/VisualizationViewHtml.java

```diff
@@ -162,6 +170,11 @@ public void setRoot(Layout layout) {
         this.noJobSelectedMessage.setWidth100();
         wrapper.addMember(noJobSelectedMessage);
 
+        this.noVisualizationMessage = new Label("The graphical Visualization is not available for this Workflow. To get one for the next execution, please open the Workflow in the Studio, and store it in the Catalog.");
```

Comment:
Do we need the Visualization with the capital letter?

Reply:
Denis likes to have capital letters, that's how he wanted the message ;)